### ISSUE TITLE
is there any way to check for the longest call youve had ( in dms or groups)

### ISSUE THREAD

**Issue by harrykotten** [peterhanania/Discord-Package#145]

**Comment by maxwellward** [comment]
Original issue title: `is there any way to check for the longest call youve had ( in dms or groups)`

Not currently in the site, no. 

The raw package has a log of all client events under `activity/analytics/events[...].json` which includes voice channel connections and disconnections along with timestamps and locations, but I'm not sure how accurate it is or how this data is logged.

It's a huge file too (mine is 4GB and millions of lines long) so it would be really hard to parse locally in a browser, but if anyone's interested in taking this on, I'd be interested in seeing how it's done. And also good luck and godspeed.
